.github/dependabot.yml

```
@@ -23,6 +23,20 @@ updates:
     assignees:
       - korobochkin
 
+  - package-ecosystem: docker
+    directories:
+      - ./development/containers/php-8.4
+    cooldown:
+      default-days: 10
+    ignore:
+      - dependency-name: "*"
+        update-types: [ "version-update:semver-patch" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)